common/src/main/java/com/alibaba/nacos/common/task/engine/TaskExecuteWorker.java

```diff
@@ -101,12 +96,12 @@ public void shutdown() throws NacosException {
      * Inner execute worker.
      */
     private class InnerWorker extends Thread {
```

Comment:
must be thread ? why not use runnable that execute by thread-pool-executor

Reply:
I just move this part, It can be enhance in other atomic PR